packages/grafana-ui/src/components/Forms/Input.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC, HTMLProps } from 'react';
+import { GrafanaTheme } from '@grafana/data';
+import { css, cx } from 'emotion';
+import { Label } from './Label';
+import { FieldValidationMessage } from './FieldValidationMessage';
+import { selectThemeVariant, stylesFactory, useTheme } from '../../themes';
+import { getFocusStyle } from './commonStyles';
+import { InputStyles } from './types';
+import { Spinner } from '..';
+import { Button } from './Button';
+
+interface Props extends HTMLProps<HTMLInputElement> {
```

Comment:
The `Input` component itself should not have any props that are label/description related. We should keep the API as simple as possible, best if it could only controll the addons, prefixes/suffixes, loading state and error state. Input should be later on composed together with labels and description in a more abstract component, i.e. `FormField`

Reply:
If we keep the label and validation message in the Input component it will become very rigid and we will have to introduce lots of points for customisation, i.e. via class names